imap_processing/common_cdf_attrs.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+global_base = {
+    "Project": ["STSP Cluster>Solar Terrestrial Science Programmes, Cluster"],
+    "Source_name": ["IMAP>Interstellar Mapping and Acceleration Probe"],
+    "Discipline": ["Solar Physics>Heliospheric Physics"],
+    "Descriptor": ["IDEX>Interstellar Dust Experiment"],
+    "PI_name": ["Dave McComas"],
+    "PI_affiliation": ["Princeton"],
+    "Instrument_type": ["Particles (space)"],
+    "Mission_group": ["IMAP"],
+}
+
+epoch_attrs = {
+    "CATDESC": "Default time",
+    "FIELDNAM": "Epoch",
+    "FILLVAL": np.array([-9223372036854775808]),
```

Comment:
```suggestion
    "FILLVAL": np.iinfo(np.int64).min,
```
Why are these an array, should they be just a single value? Also might help to say it is the minimum possible int64?

Same for the other values below this.